web/react/components/edit_channel_modal.jsx

```diff
@@ -6,17 +6,25 @@ var AsyncClient = require('../utils/async_client.jsx');
 
 module.exports = React.createClass({
     handleEdit: function(e) {
-        var data = {}
+        var self = this;
```

Comment:
Yeah I don't see why not, I think I added it on reflex due to having the jQuery call near it, let me take it out and test to be sure though.